control-plane/api/v1alpha1/exportedservices_types.go

```diff
@@ -73,6 +74,8 @@ type ServiceConsumer struct {
 	Partition string `json:"partition,omitempty"`
 	// [Experimental] Peer is the name of the peer to export the service to.
```

Comment:
I know this isn't you, but can you verify if Peering is still `experimental` for K8s? and if not change this?